website/src/content/concepts/oauth-scopes-and-user-permissions.mdx

```diff
@@ -30,19 +30,67 @@ Notice here how the OAuth Scopes are grouped by the two fields `view` and `manag
 
 This grouping determines the **mapping and relation between OAuth Scopes and user permissions**.
 
+## Permission groups
+
+Every Custom Application has a **default permission group** with a pair of view and manage permissions. This group maps to the OAuth Scopes specified in the [oAuthScopes](/api-reference/application-config#oauthscopes) field of the Custom Application config.
+
+However, you might need more granular access control to fulfil specific business requirements. For example, if your Custom Application manages products, discounts, and orders, you might want only a group of users to manage products and discounts while another group of users handles orders.<br />
+In this kind of scenario a single permission group (the default one) is not enough as you would need multiple permission groups fulfil the access requirements.
+
+To enable such use cases, you can define additional permission groups that have different access requirements. See [additionalOAuthScopes](/api-reference/application-config#additionaloauthscopes) field in the Custom Application config. 
+
+<Info>
+
+This feature is available from version `21.21.0` onwards.
+
+</Info>
+
+In the following example, we are defining two additional permission groups. The group `delivery` allows users to manage incoming orders while the group `promotion` enables users to work on discount and promotion campaigns.
+
+```json title="custom-application-config.json" highlightLines="6-17"
+{
+  "oAuthScopes": {
+    "view": ["view_products", "view_customers"],
+    "manage": ["manage_products"]
+  },
+  "additionalOAuthScopes": [
+    {
+      "name": "delivery",
+      "view": [],
+      "manage": ["manage_orders"]
+    },
+    {
+      "name": "promotion",
+      "view": [],
+      "manage": ["manage_orders", "manage_discount_codes"]
+    }
+  ]
+}
+```
+
+<Info>
+
+The default permission group is always defined, even when adding additional groups.
+
+When additional groups are defined, the default group can be left "empty," without specifying any OAuth Scopes.
+
+Even so, remember that the "view-only" user permission must be enabled to allow access to the Custom Application.
```

Comment:
Hmm here we're actually mixing things up a bit. The first part about defining the permission groups is about the Custom Application configuration while the "view-only" part is about assigning team permissions.

Having the two points in one sentence makes is look like they are strictly related when configuring the app.

Maybe you can try rephrasing this again in case the original version wasn't clear enough?

Reply:
@emmenko, thanks for highlighting that. I've rephrased it for clarity.